src/templates/api_reference/resource.ejs

```diff
@@ -0,0 +1,70 @@
+<%
+var thisType = plural(cap(resource.name));
+
+%>---
+parent-title: API Reference
+title: <%= thisType %>
+description:
+_template: dev-api-reference
+_layout: dev
+
+
+# Sidebar nav
+side_nav:
+  <% _.forEach(resource.action_classes, function(action_class) {%><%= '-'%>
+    <%= 'name: ' + action_class.name%>
+    <%= 'url: ' + action_class.url%>
+<%});%>
+
+# Header
+logo_text: Developers
+header_type: navigation
+header_search: true;
+---
+
+### <%= thisType%>
+
+<%= removeLineBreaks(resource.comment)%>
+<%if (resource.notes) {%><% _.forEach(resource.notes, function(note) {%>
+<%= removeLineBreaks(note)%>
+<%})};%>
+
+<%= thisType%> have the following fields:
+
+| Field | Description |
+|---|---|<% _.forEach(resource.properties, function(property) {%>
+<%= '| ' + property.name + ' | '%><%_.forEach(property.example_values, function(value) {
+  %><%= '<code class="table-example">' + value + '</code> '%><%});%><%if (property.access) {
+%><%= '**' + property.access + '.** '%><%}%><%= removeLineBreaks(property.comment)%><%if (property.notes) {
+%>**Note:** <% _.forEach(property.notes, function(note) {%><%= removeLineBreaks(note)%><%})};%>|<%});%>
+
+<% _.forEach(resource.action_classes, function(action_class) {
+  var matchingActions = resource.actions.filter(function(action) {
+    return action.class == action_class.url;
+  });
+%><%= '<a name="' + action_class.url + '"></a>'%>
```

Comment:
Yeah, you have a lot of string constants in here. It should be nicer to pull them out into the template itself.